lib/Search/SqlSearch/SimpleSqlSearchEngine.php

```diff
@@ -83,16 +89,16 @@ public function search(SearchQuery $query)
      *
      * @return array
      */
-    protected function filterTableStructure(array $fields)
+    protected function filterTableStructure(array $fields): array
     {
         $filteredFields = [];
 
         foreach ($fields as $name => $type) {
-            if (strpos($type, 'varchar') == 0) {
+            if (str_starts_with($type, 'varchar')) {
                 $filteredFields[$name] = 'varchar';
             }
 
-            if (strpos($type, 'text') == 0) {
+            if (str_starts_with($type, 'text')) {
```

Comment:
Hi @Dillon-Brown I think `str_starts_with` its only supported in php8

Reply:
We have polyfills for these, I'm going to revert anyway though as I believe the last Suite-7 release we had some issues with the polyfills when it came to running the upgrade wizard. Nice spot.